### DISCUSSION TITLE
I am having this error code FST_ERR_CTP_BODY_TOO_LARGE after increasing bodyParser limit

### DISCUSSION THREAD

**Discussion by dzeroone** [vercel/next.js#28365] [Help]
this is the config for one of my api route
```
export const config = {
  api: {
    bodyParser: {
      sizeLimit: '50mb',
    },
  },
}
```
i am getting this error response when uploading a gif image
```
{"statusCode":413,"code":"FST_ERR_CTP_BODY_TOO_LARGE","error":"Payload Too Large","message":"Request body is too large"}
```
I can't find any solution for this problem. Please suggest something.

**Comment by zneb** [comment]
This looks like an error from Fastify, you might be using a custom server. If you are, try also increasing the body size limit from there. [Fastify Docs](https://www.fastify.io/docs/latest/Server/#bodylimit)

**Reply by dzeroone** [reply]
sorry my mistake, it was coming from api.
